test/integration/performance/redis_test.go

```diff
@@ -0,0 +1,241 @@
+//go:build integration || performance
+// +build integration performance
+
+package performance
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/skupperproject/skupper/test/integration/performance/common"
+	"github.com/skupperproject/skupper/test/utils/base"
+	"github.com/skupperproject/skupper/test/utils/k8s"
+	"gotest.tools/assert"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ENV_REDIS_REQUESTS = "REDIS_NUMBER_REQUESTS"
+	ENV_REDIS_CLIENTS  = "REDIS_PARALLEL_CLIENTS"
+	ENV_REDIS_TIMEOUT  = "REDIS_TIMEOUT"
+	ENV_REDIS_CPU      = "REDIS_CPU"
+	ENV_REDIS_MEMORY   = "REDIS_MEMORY"
+)
```

Comment:
Maybe for a future enhancement, add a variable to specify a comma separated list of tests to run. Empty list would run "all"